ngraph/core/include/ngraph/pass/convert_fp32_to_fp16.hpp

```diff
@@ -10,21 +10,11 @@ namespace ngraph
 {
     namespace pass
     {
-        class NGRAPH_API ConvertFP32ToFP16 : public ngraph::pass::GraphRewrite
+        class NGRAPH_API ConvertFP32ToFP16 : public ngraph::pass::FunctionPass
```

Comment:
What is about the backward compatibility?
What if somebody already uses this path as GraphRewrite?

Reply:
As GraphRewrite inherits FunctionPass and has the same API so there is no any backward compatibility issues.